### PR TITLE
Stacked area chart tooltip

### PR DESCRIPTION
**Pull request by ozankaraali**

### CODE REVIEW

File: src/components/stacked-area-chart/StackedAreaChart.tsx [hunk 1]
```diff
@@ -156,88 +161,143 @@ function maxLengthOfAllArrays(series: Series[]): number {
...
-                    y2={marY + chartY}
-                    stroke="black"
-                    strokeWidth="4" 
+export class StackedAreaChart extends React.Component<StackedAreaChartProps, StackedAreaChartState> {
+    state = {
+        hoverId: 0
+    };
+
+    copy(series: Series[]): Series[] {
+        return JSON.parse(JSON.stringify(series));
```
Comment (Samoxive) [inline src/components/stacked-area-chart/StackedAreaChart.tsx]: ಠ_ಠ

File: src/components/stacked-area-chart/StackedAreaChart.tsx [hunk 2]
```diff
@@ -156,88 +161,143 @@ function maxLengthOfAllArrays(series: Series[]): number {
...
+                    type="light"
+                    effect="float"
+                    id="stacked-area-chart-tooltip"
+                    className="chart-tooltip"
+                >
+                    {hoverId !== -1 ? (
+                        <div>
+                            {series.map((num, i) =>
+                            <div key={i}>{series[i].label + ': ' + series[i].data[hoverId]}</div>
+                            )
```
Comment (Samoxive) [inline src/components/stacked-area-chart/StackedAreaChart.tsx]: Just minor nitpick, you can put `)}` on the same line and indent `<div>`.

**Review by Samoxive — CHANGES_REQUESTED** [review]
Good work.

File: src/components/stacked-area-chart/StackedAreaChart.tsx (hunk 1 above)
Comment (ozankaraali) [reply src/components/stacked-area-chart/StackedAreaChart.tsx]: yep, there is some copying, as you said. ( ͡° ͜ʖ ͡°)

File: src/components/stacked-area-chart/StackedAreaChart.tsx (hunk 2 above)
Comment (ozankaraali) [reply src/components/stacked-area-chart/StackedAreaChart.tsx]: Okay, let me.

**Pull request merged**